config/initializers/gds_sso.rb

```diff
@@ -1,10 +1,3 @@
-GDS::SSO.config do |config|
-  config.user_model   = "User"
-  config.oauth_id     = ENV["OAUTH_ID"]
-  config.oauth_secret = ENV["OAUTH_SECRET"]
-  config.oauth_root_url = Plek.new.external_url_for("signon")
-end
-
 if Rails.env.development?
```

Comment:
😱 weird place for a seed :-/

Reply:
Yeah, it surprised me as well. I thought about it a bit more and decided to add another commit. We still can't remove the initializer entirely, due to the behaviour of the gem.

Reply:
Could you take another look?

Reply:
Oh nice one Ben - didn't expect you to follow up on this ⭐